src/components/Form/FormFactory.js

```diff
@@ -98,7 +98,7 @@ export const FormFactory = ({
           </Col>
           <Col sm={controlSize}>
             {child}
-            <HelpBlock>{validMsg}</HelpBlock>
+            <HelpBlock>{valid ? valid.message : null}</HelpBlock>
```

Comment:
shouldn't this be invalid? (assuming its a validation error hence its not valid?)

Reply:
validation can return warning or info too. So the result does not always has to erroneous. So the `errors` does not really fit here IMHO.

Reply:
do you have examples in mind? the only usage case I could think of would be a weak password? regardless, in that case, we should have messages that are errors, and others that are info/warn? in sounds to me we need to know which level the field validation are, so we can highlight in red / yellow the entire field for errors?

Reply:
example is here https://github.com/kubevirt/web-ui-components/pull/105#discussion_r234536393 , the message is actually highlighted red/yellow/grey (error/warning/info) by https://github.com/kubevirt/web-ui-components/pull/105/files#diff-a64432c062870774c8559d8af0c30ff7R88 (from pf-react)

Reply:
renamed `valid` to `validation`